### ISSUE TITLE
claimData not match

### ISSUE THREAD

**Issue by juniuszhou** [Sifchain/sifnode#2235]
In future/peggy2 branch, the claimData not match between smart contract and ebrelayer.
we also need store missed items in prophecy info in sifnode.

// smart contract
struct ClaimData {
bytes cosmosSender;
uint256 cosmosSenderSequence;
address payable ethereumReceiver;
address tokenAddress;
uint256 amount;
string tokenName;
string tokenSymbol;
uint8 tokenDecimals;
int32 networkDescriptor;
bool doublePeg;
uint128 nonce;
string cosmosDenom;
}

// CosmosBridgeClaimData is an auto generated low-level Go binding around an user-defined struct.
type CosmosBridgeClaimData struct {
CosmosSender []byte
CosmosSenderSequence *big.Int
EthereumReceiver common.Address
TokenAddress common.Address
Amount *big.Int
DoublePeg bool
Nonce *big.Int
NetworkDescriptor *big.Int
TokenName string
TokenSymbol string
TokenDecimals uint8
}

**Comment by daniportugalgit** [comment]
There were many issues.
1) The script had been modified a long time ago in a way that it stopped generating .go files. We didn't realize it earlier because we spent a long time working on different branches.
=> When we fixed that, abigen started throwing errors when executed.

2) As James reported here (https://sifchain.slack.com/archives/C02JAGKD04B/p1638188875082800), "Part the problem is that we have abigen code that depends on an obsolete version of abigen; I'm fixing that now".

Also, James is deprecating our old script, because it did more than what we needed, and updating our `make` flow so that abigen runs automatically when we need it.

Therefore, this task is done and James's task (https://app.zenhub.com/workspaces/current-sprint---engineering-615a2e9fe2abd5001befc7f9/issues/sifchain/sifnode/2233) will finalize the issue and solve our problems with abigen.
